### PR TITLE
Use HTTP typed actions and idiomatic C#

### PR DESCRIPTION
**Pull request by pBouillon**
Hey!

Just stumbled across this project and saw a couple of things to improve:

- Missing solution file
- `.gitignore` with missing entries
- Malformed log templates
- Unused dependencies
- Violations of C# recommandations (multiple classes per file, etc.)
- Raw objects returned instead of the associated HTTP object

Hope this will help!

### CODE REVIEW

File: dotnet-6/Models/User.cs [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    }
-}
\ No newline at end of file
+namespace CrudApp.Models;
+
+public class User
+{
+    public long Id { get; set; }
+    public string Email { get; set; } = string.Empty;
+    public string Name { get; set; } = string.Empty;
```
Comment (manishrw) [inline dotnet-6/Models/User.cs]: Why do we need to initialize them to empty when there is no empty constructor?

File: dotnet-6/Services/IUserService.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using CrudApp.Models;
+
+namespace CrudApp.Services;
+
+public interface IUserService
+{
+    IEnumerable<User> GetUsers();
+
```
Comment (manishrw) [inline dotnet-6/Services/IUserService.cs]: nit pick: remove empty lines

**Review by manishrw — CHANGES_REQUESTED** [review]

**Comment by manishrw** [comment]
Good job on creating this PR. Have put some minor comments, we can merge the PR after they are addressed. Thanks!

File: dotnet-6/Models/User.cs (hunk 1 above)
Comment (pBouillon) [reply dotnet-6/Models/User.cs]: We do not, indeed, I removed that 😄

**Comment by pBouillon** [comment]
Thanks for the review!  
I addressed the remarks you made, let me know if that's good enough for you 😄

**Pull request merged**